homeassistant/components/deconz/device_trigger.py

```diff
@@ -300,6 +300,78 @@
     (CONF_QUADRUPLE_PRESS, CONF_TURN_ON): {CONF_EVENT: 1006},
 }
 
+AQARA_OPPLE_2_BUTTONS_MODEL = "lumi.remote.b286opcn01"
+AQARA_OPPLE_2_BUTTONS = {
+    (CONF_LONG_PRESS, CONF_TURN_OFF): {CONF_EVENT: 1001},
+    (CONF_SHORT_RELEASE, CONF_TURN_OFF): {CONF_EVENT: 1002},
+    (CONF_LONG_RELEASE, CONF_TURN_OFF): {CONF_EVENT: 1003},
+    (CONF_DOUBLE_PRESS, CONF_TURN_OFF): {CONF_EVENT: 1004},
+    (CONF_TRIPLE_PRESS, CONF_TURN_OFF): {CONF_EVENT: 1005},
+    (CONF_LONG_PRESS, CONF_TURN_ON): {CONF_EVENT: 2001},
+    (CONF_SHORT_RELEASE, CONF_TURN_ON): {CONF_EVENT: 2002},
+    (CONF_LONG_RELEASE, CONF_TURN_ON): {CONF_EVENT: 2003},
+    (CONF_DOUBLE_PRESS, CONF_TURN_ON): {CONF_EVENT: 2004},
+    (CONF_TRIPLE_PRESS, CONF_TURN_ON): {CONF_EVENT: 2005},
+}
+
+AQARA_OPPLE_4_BUTTONS_MODEL = "lumi.remote.b486opcn01"
+AQARA_OPPLE_4_BUTTONS = {
+    (CONF_LONG_PRESS, CONF_TURN_OFF): {CONF_EVENT: 1001},
+    (CONF_SHORT_RELEASE, CONF_TURN_OFF): {CONF_EVENT: 1002},
+    (CONF_LONG_RELEASE, CONF_TURN_OFF): {CONF_EVENT: 1003},
+    (CONF_DOUBLE_PRESS, CONF_TURN_OFF): {CONF_EVENT: 1004},
+    (CONF_TRIPLE_PRESS, CONF_TURN_OFF): {CONF_EVENT: 1005},
+    (CONF_LONG_PRESS, CONF_TURN_ON): {CONF_EVENT: 2001},
+    (CONF_SHORT_RELEASE, CONF_TURN_ON): {CONF_EVENT: 2002},
+    (CONF_LONG_RELEASE, CONF_TURN_ON): {CONF_EVENT: 2003},
+    (CONF_DOUBLE_PRESS, CONF_TURN_ON): {CONF_EVENT: 2004},
+    (CONF_TRIPLE_PRESS, CONF_TURN_ON): {CONF_EVENT: 2005},
+    (CONF_LONG_PRESS, CONF_DIM_DOWN): {CONF_EVENT: 3001},
+    (CONF_SHORT_RELEASE, CONF_DIM_DOWN): {CONF_EVENT: 3002},
+    (CONF_LONG_RELEASE, CONF_DIM_DOWN): {CONF_EVENT: 3003},
+    (CONF_DOUBLE_PRESS, CONF_DIM_DOWN): {CONF_EVENT: 3004},
+    (CONF_TRIPLE_PRESS, CONF_DIM_DOWN): {CONF_EVENT: 3005},
+    (CONF_LONG_PRESS, CONF_DIM_UP): {CONF_EVENT: 4001},
+    (CONF_SHORT_RELEASE, CONF_DIM_UP): {CONF_EVENT: 4002},
+    (CONF_LONG_RELEASE, CONF_DIM_UP): {CONF_EVENT: 4003},
+    (CONF_DOUBLE_PRESS, CONF_DIM_UP): {CONF_EVENT: 4004},
+    (CONF_TRIPLE_PRESS, CONF_DIM_UP): {CONF_EVENT: 4005},
```

Comment:
I would like something like this
```suggestion
    **AQARA_OPPLE_2_BUTTONS,
    (CONF_LONG_PRESS, CONF_DIM_DOWN): {CONF_EVENT: 3001},
    (CONF_SHORT_RELEASE, CONF_DIM_DOWN): {CONF_EVENT: 3002},
    (CONF_LONG_RELEASE, CONF_DIM_DOWN): {CONF_EVENT: 3003},
    (CONF_DOUBLE_PRESS, CONF_DIM_DOWN): {CONF_EVENT: 3004},
    (CONF_TRIPLE_PRESS, CONF_DIM_DOWN): {CONF_EVENT: 3005},
    (CONF_LONG_PRESS, CONF_DIM_UP): {CONF_EVENT: 4001},
    (CONF_SHORT_RELEASE, CONF_DIM_UP): {CONF_EVENT: 4002},
    (CONF_LONG_RELEASE, CONF_DIM_UP): {CONF_EVENT: 4003},
    (CONF_DOUBLE_PRESS, CONF_DIM_UP): {CONF_EVENT: 4004},
    (CONF_TRIPLE_PRESS, CONF_DIM_UP): {CONF_EVENT: 4005},
```